_posts/2020-08-01-Fortran-Newsletter-August-2020.md

```diff
@@ -0,0 +1,158 @@
+  books about Fortran
+
+* [#121](https://github.com/fortran-lang/fortran-lang.org/pull/121), [#122](https://github.com/fortran-lang/fortran-lang.org/pull/122), [#127](https://github.com/fortran-lang/fortran-lang.org/pull/127), [#128](https://github.com/fortran-lang/fortran-lang.org/pull/128): additional packages added to the Fortran-lang.org [packages](https://fortran-lang.org/packages) page
+  
+Ongoing work:
+
+* [#117](https://github.com/fortran-lang/fortran-lang.org/issues/117): Adding a
+  Benchmarks section, a new dedicated repository was created at
+  https://github.com/fortran-lang/benchmarks and many details have been
+  discussed in [issues](https://github.com/fortran-lang/benchmarks/issues) there
+
+
+
+[Let us know](https://github.com/fortran-lang/fortran-lang.org/issues) if you have any suggestions for the website and its content.
+We welcome any new contributors to the website and the tutorials page in particular - see the [contributor guide](https://github.com/fortran-lang/fortran-lang.org/blob/master/CONTRIBUTING.md) for how to get started.
+
+
+## Fortran Standard Library
+
+What's new in the Fortran Standard Library:
+[#223](https://github.com/fortran-lang/stdlib/pull/223): the structure of the Fortran Standard Library has been modified for clarity and ease of use. With these changes, both experimental and stable procedures will reside together in the same modules. The status of the procedures (experimental vs stable) are documented in the code, in the specs, and in the [API docs](https://stdlib.fortran-lang.org/)
+
+Main ongoing discussions:
+* [#225](https://github.com/fortran-lang/stdlib/issues/225): Name convention for derived types in `stdlib`
+* [#224](https://github.com/fortran-lang/stdlib/issues/224): Handling and propagating errors inside `stdlib`
+* [#221](https://github.com/fortran-lang/stdlib/issues/221): API for a bitset data type
+* [#201](https://github.com/fortran-lang/stdlib/issues/201): API for file system operations
+
+## Fortran Package Manager
+
+What's new in fpm:
+* We created the [fpm-registry](https://github.com/fortran-lang/fpm-registry) repository,
+which serves as a registry of fpm-enabled Fortran packages.
+Please see the README there to learn how to contribute a package.
+For now, the registry is simply a list of fpm-enabled Fortran packages that you can use as a dependency in your `fpm.toml` file.
+Soon, this registry will be used to generate detailed metadata that will be used by fpm to allow you to search for packages from the command-line, e.g. `fpm search <package>` or similar.
+* [#146](https://github.com/fortran-lang/fpm/issues/146): We discussed the design of the new Fortran implementation of fpm in a video call. We agreed on the need for an intermediate package model which will allow for clean separation of fpm frontends (user interface, parsing, and semantics) and fpm backends (fpm itself, CMake, Make, etc.).
+
```

Comment:
```suggestion
* [#131](https://github.com/fortran-lang/fpm/pull/131),
[#132](https://github.com/fortran-lang/fpm/pull/132),
[#139](https://github.com/fortran-lang/fpm/pull/139),
[#140](https://github.com/fortran-lang/fpm/pull/140),
[#142](https://github.com/fortran-lang/fpm/pull/142),
[#145](https://github.com/fortran-lang/fpm/pull/145),
[#147](https://github.com/fortran-lang/fpm/pull/147),
[#148](https://github.com/fortran-lang/fpm/pull/148),
[#151](https://github.com/fortran-lang/fpm/pull/151):
We merged several pull requests toward the Fortran fpm implementation. The Haskell implementation has moved to the `fpm/bootstrap` directory, and the Fortran implementation is developed in `fpm/fpm`. The Fortran fpm is, of course, an fpm package itself so it can be built by the Haskell fpm. Soon, it will be able to be build by itself. 

```